src/analysis/inner_product_space/projection.lean

```diff
@@ -612,6 +612,11 @@ variables (K)
 /-- Reflection is involutive. -/
 lemma reflection_involutive : function.involutive (reflection K) := reflection_reflection K
 
+/-- Reflection is involutive. -/
+@[simp] lemma reflection_trans_reflection :
+  (reflection K).trans (reflection K) = linear_isometry_equiv.refl 𝕜 E :=
+by ext; simp
```

Comment:
This probably can become 
```suggestion
ext reflection_involutive
```
Although ext might need namespacing

Reply:
I couldn't get this to work because of complex-number `ext` breaking a goal into real and imaginary parts -- did you try it?

Reply:
No, I'll try it tomorrow :)

Reply:
Fixed - it's not shorter, but it saves simp searching for a lemma one line above.